syne_tune/optimizer/baselines.py

```diff
@@ -375,6 +420,24 @@ def __init__(
         )
 
 
+def _create_searcher_kwargs(
+    config_space: dict,
+    metric: str,
+    random_seed: Optional[int],
+    kwargs: dict,
+) -> dict:
+    searcher_kwargs = dict(
+        config_space=config_space,
+        metric=metric,
+        points_to_evaluate=kwargs.get("points_to_evaluate"),
+    )
+    search_options = kwargs.get("search_options", dict())
+    searcher_kwargs.update(search_options)
+    if random_seed is not None:
```

Comment:
This is how random_seed for searcher relates to overall random seed passed to scheduler, if the searcher is created up front